### DISCUSSION TITLE
Allow Read/Write Access for Chat GPT to make changes with in selected [[Linked]] Notes on a User request.

### DISCUSSION THREAD

**Discussion by captorxxv** [brianpetro/obsidian-smart-connections#286] [Ideas]
Just an idea, but it would be great if user can request to make a changes with in a selected note so AI would make changes, add content or task list in markdown format based on user chat request 
Not sure if this would be possible ?

**Comment by captorxxv** [comment]
Another way, would be automaticly dumping some of the chat content in to a draft note so user can than copy selected content from this note in to selected location etc..

**Comment by brianpetro** [comment]
Yes, this is something I'm working on.
